reana-ui/src/pages/signin/Signin.js

```diff
@@ -28,14 +28,14 @@ export default function Signin() {
   const [formData, setFormData] = useState({ email: "", password: "" });
   const location = useLocation();
 
-  const handleClick = () => {
+  const handleClick = (url) => {
```

Comment:
I think we should change the name of `url` to something more descriptive, what do you think about `ssoProvider` or something similar?